apps/pie-docs/src/assets/js/show-more.js

```diff
@@ -1,29 +1,27 @@
 let shown = false;
 let showMoreButtons;
 
-const initialiseShowMore = () => {
+window.addEventListener('DOMContentLoaded', () => {
     showMoreButtons = document.querySelectorAll('[data-js="show-more"]');
     const componentStatusTables = document.querySelectorAll('[class="c-componentStatusTable "]');
 
-    if (!showMoreButtons) return;
-
     // ensures status tables initially only show 6 columns each
-    if (componentStatusTables) {
-        componentStatusTables.forEach((el) => {
-            el.querySelectorAll('tbody tr').forEach((tr, index) => {
-                const rows = index >= 6 ? tr.style.display = 'none' : '';
+    componentStatusTables.forEach((el) => {
```

Comment:
nit: I would add an early return if the length of showMoreButtons or componentStatusTables before the `forEach` just to make sure the script doesn't necessarily run for other pages